src/topology/continuous_function/algebra.lean

```diff
@@ -113,6 +113,27 @@ instance {α : Type*} {β : Type*} [topological_space α] [topological_space β]
   ..continuous_map.semigroup,
   ..continuous_map.has_one }
 
+@[to_additive]
+instance {α : Type*} {β : Type*} [topological_space α]
+  [locally_compact_space α] [topological_space β]
+  [monoid β]  [has_continuous_mul β] : has_continuous_mul C(α, β) :=
+⟨begin
+  refine continuous_of_continuous_uncurry _ _,
+  change continuous ((λ (x : β × β), x.1 * x.2) ∘
+    (λ (x : (C(α, β) × C(α, β)) × α), (x.1.1 x.2, x.1.2 x.2))),
+  have h1 : continuous (λ (x : (C(α, β) × C(α, β)) × α), (x.fst.fst) x.snd),
+  { change continuous ((λ (x : C(α, β) × α), x.1 x.2) ∘
+      (λ (x : (C(α, β) × C(α, β)) × α), (x.1.1, x.2))),
+    refine continuous_ev.comp ((continuous_fst.comp continuous_fst).prod_mk continuous_snd), },
+  have h2 : continuous (λ (x : (C(α, β) × C(α, β)) × α), (x.fst.snd) x.snd),
+  { change continuous ((λ (x : C(α, β) × α), x.1 x.2) ∘ (λ (x : (C(α, β) × C(α, β)) × α),
+      (x.1.2, x.2))),
+    refine continuous_ev.comp ((continuous_snd.comp continuous_fst).prod_mk continuous_snd), },
+  refine (continuous_fst.comp (continuous.prod_mk _ _)).mul
+    (continuous_snd.comp (continuous.prod_mk _ _)),
+  all_goals { assumption, },
```

Comment:
```suggestion
  exact (continuous_fst.comp (h1.prod_mk h2)).mul (continuous_snd.comp (h1.prod_mk h2)),
```
You can do something similar in the other proof.

Reply:
Done